### PR TITLE
feat: fixed and added more shard events

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/events/shard/shardReady.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ShardEvent } from '@lib/structures/ShardEvent';
+import { green } from 'colorette';
+
+export default class extends ShardEvent {
+	protected readonly title = green('Ready');
+
+	public run(id: number, unavailableGuilds: Set<string> | null) {
+		this.client.console.error(`${this.header(id)}: ${unavailableGuilds?.size ?? 'Unknown'} Guilds`);
```
Comment (vladfrangu) [inline src/events/shard/shardReady.ts]:
```suggestion
		this.client.console.error(`${this.header(id)}: ${unavailableGuilds?.size ?? 'Unknown or no unavailable'} guilds`);
```

File: src/events/shard/shardError.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ShardEvent } from '@lib/structures/ShardEvent';
+import { red } from 'colorette';
+
+export default class extends ShardEvent {
+	protected readonly title = red('Error');
+
+	public run(error: Error, id: number) {
+		this.client.console.error(`${this.header(id)}: ${error.message}`);
```
Comment (vladfrangu) [inline src/events/shard/shardError.ts]:
```suggestion
		this.client.console.error(`${this.header(id)}: ${error.stack}`);
```

File: src/events/shard/shardDisconnect.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { ShardEvent } from '@lib/structures/ShardEvent';
+import { red } from 'colorette';
+
+export default class extends ShardEvent {
+	protected readonly title = red('Disconnected');
+
+	public run(event: number, id: number) {
```
Comment (vladfrangu) [inline src/events/shard/shardDisconnect.ts]: `event` is not a number, but a CloseEvent

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: src/events/shard/shardReady.ts (hunk 1 above)
Comment (kyranet) [reply src/events/shard/shardReady.ts]: Resolved all requested changes in 027243f6.

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**